tests/Unit/Time/StepControllers/Test_FullSlab.cpp

```diff
@@ -15,7 +15,7 @@
 #include "Time/Time.hpp"
 
 SPECTRE_TEST_CASE("Unit.Time.StepControllers.FullSlab", "[Unit][Time]") {
-  Parallel::register_derived_classes_with_charm<StepController>();
+  Parallel::register_classes_in_list<tmpl::list<StepControllers::FullSlab>>();
```

Comment:
Do you think a `Parallel::register_derived_class<Class>()` function would be useful for these situations?

Reply:
Not sure about something specialized to a single class, but a more general version using a variadic template instead of a list has some appeal to it.